Resources/Prototypes/Research/industrial.yml

```
@@ -146,6 +145,20 @@
   - HolofanProjector
   - PortableScrubberMachineCircuitBoard
 
+- type: technology
+  id: AdvancedToolsTechnology
+  name: research-technology-advanced-tools
+  icon:
+    sprite: Objects/Tools/drill.rsi
+    state: drill_bolt
+  discipline: Industrial
+  tier: 2
+  cost: 15000
```

```suggestion
  cost: 10000
```
No need for it to be that high imo. The only really powerful thing is the JoL.